wherebuilder/qb.py

```diff
@@ -41,7 +55,11 @@ def _format_with_args(self, query_stmt, matches, *args):
 
     def _format_with_kwargs(self, query_stmt, matches, **kwargs):
         for m in matches:
-            query_stmt = query_stmt.replace(m, self._value_by_type(kwargs.get(m[1:], '')))
+            query_stmt = query_stmt.replace(m, '{' + m[1:] + '}')
+
+        for key in kwargs:
+            kwargs[key] = self._value_by_type(kwargs[key])
+        query_stmt = query_stmt.format(**kwargs)
```

Comment:
I think these are redundant because it's enough to replace with the previous way in order of the sorted `matches`.